### ISSUE TITLE
[CAIA A11y Research - PILOT] Veteran-Facing Forms: Form Upload

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#85573]
> [!IMPORTANT]
> - **TICKET STATUS:** `Scheduled`
> - **PILOT DATE:**  6/18/2024 11 a.m. to noon (Eastern)
> - **TYPES OF ASSISTIVE TECH:** Screen Reader `Specific AT not yet known`

# CAIA A11y Support
This is a child ticket of the research ticket [85447](https://github.com/department-of-veterans-affairs/va.gov-team/issues/85447) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

@SarahKay8 
@sara-amanda 

</details>

###  About this  VFF Research

> The Veteran-facing forms team would like to evaluate a new tool that will allow all authenticated Veterans, including those using AT, to access and upload forms within the modernized VA.gov eco-system.

> Based on the outcome of this research study, we’ll either coordinate with sitewide to release our MVP to production or update our design to correct any significant usability hurdles people encounter.

<details><summary>Toggle to view team hypothesis and methodology</summary>

**Info Source:** [Extracted for quick reference from the VFF research plan.](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/DocumentUpload_FindaForm/Research%20/Research%20plan.md)

#### Hypothesis
> - Form Upload will meet participants' expectations
> - Most participants will already be familiar with how to find a specific form on VA.gov
> - Some, but not all Veterans will be familiar with QuickSubmit
> - Of the participants who are familiar with QuickSubmit, the majority will prefer Form Upload to QuickSubmit
> - Most participants will prefer to upload their form using Form Upload vs. existing alternatives

#### Methodology
> - Remote moderated usability tests with Veterans, including AT users and users with cognitive impairments.
> - We’ll test with a coded prototype in staging.
> - Participants will download + upload the form PDF from the Form's detail page
> - To access the form upload flow, we’ll provide participants with credentials for test accounts.
> - After each session, we’ll update passwords so participants can’t re-authenticate with the same credentials.

</details>

### Team Contacts
<details><summary>Toggle to view team info</summary>

- Team name: Veteran Facing Forms
- OCTO-DE product owner: Matthew Dingee
- Product manager: Tricia Baker
- Designer: Rachel Shearer
- FE engineer: Eric Tillberg
- Product/team Slack channel: #veteran-facing-forms
- Accessibility champion: Jeana Clark

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session
- **Date:** `TBD` (Goal between 6/17 and 6/21/2024)
- **Prototype**: Coded prototype in Staging
- **[Research Convo Guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/DocumentUpload_FindaForm/Research%20/Conversation%20guide.md)**
- **[Research Plan](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/DocumentUpload_FindaForm/Research%20/Research%20plan.md)**

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Share link to [Assistive Tech Pilot Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/assistive-tech-pilot-guide.md) with team, if they haven't reviewed it yet**
- [x] Hold added to calendar for timespan 6/17-6/21/2024 for pilot.
- [x] **Schedule CAIA a11y:** Provided availability on 6/11/2024 - ss
- [x] **Research Tracker** Holds Added 6/11/2024 - ss
- [x] **Update research outlook tracker** with official dates
- [x] **Determine date and time for pilot:** 6/18/2024 11 - 12 p.m. with SK
- [x] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/DocumentUpload_FindaForm/Research%20/Conversation%20guide.md) and provide feedback in comments.
- [x] Request AT device and tech to be used in sessions - **JAWS**
- [x] [Receive AT device and tech from Team in research ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/85573#issuecomment-2161462954) to be used in sessions to prep for pilot
- [x] **Review prototype** in `Location` - `STATUS OF PROTOTYPE`
- [x] **Run pilot** with VFS Researcher
- [x] **Optional: Note Taking:** If there is only one a11y, notes cannot be taken due to pilot participation. Folder for Team in GH Deliverables for CAIA A11Ys.
```

**Comment by NaomiPMC** [comment]
@sara-amanda  can I move this to ready, looks like you guys are underway with planning?

**Comment by tbaker1026** [comment]
@sara-amanda I am the Product Manager on this research, Rachel is out this week but wanted to check to see if I needed to schedule an AT Pilot session for next week?  Thank you

**Comment by sara-amanda** [comment]
> @sara-amanda I am the Product Manager on this research, Rachel is out this week but wanted to check to see if I needed to schedule an AT Pilot session for next week? Thank you

Hi, @tbaker1026 can you send me over some time slots that work best for you, and your team so that  we can schedule the pilot? 

Take a look at the **[CAIA Accessibility Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?height=1000&wkst=1&ctz=America/New_York&bgcolor=%23B39DDB&src=Y19mOTgyMzUwZmM3M2E4MTYyYTEwODhhYjE0ZDAwZGQzNzMzNWIyYjU1OWE2MGQ2YjY3ZWI4MDIxNDE2ZjgxYzJhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20&color=%23616161)** and let us know what works for you? We have blocked off times that team members are not available for research, or are conducting other research. 

Next Week
- We are all out of office on 6/19/2024 for the holiday. 
- Limited availability on Friday. 
- Monday, 9-12 appears to be free
- Tuesday 9-12 appears to be free
-

**Comment by tbaker1026** [comment]
@sara-amanda Are those times Eastern?

**Comment by sara-amanda** [comment]
> @sara-amanda Are those times Eastern? If yes can we do Tuesday from 11-12?

@SarahKay8 will be conducting the pilot and is available. 

@tbaker1026 Just a few questions: 
- Who would you like me to include on the invite? 
- What tech would you prefer we use for the pilot testing? I see screen reader, but do you have a specific one in mind?

**Comment by tbaker1026** [comment]
@sara-amanda 
rshearer@pluribusdigital.com
tricia.baker@oddball.io
jaci.wilkinson@adhocteam.us

Lets us JAWS

**Comment by tbaker1026** [comment]
@SarahKay8 Do you want me to set up a meeting?

**Comment by sara-amanda** [comment]
> @sara-amanda [rshearer@pluribusdigital.com](mailto:rshearer@pluribusdigital.com) [tricia.baker@oddball.io](mailto:tricia.baker@oddball.io) [jaci.wilkinson@adhocteam.us](mailto:jaci.wilkinson@adhocteam.us)
> 
> Lets use JAWS

@tbaker1026 and @SarahKay8 : 
- [x] The invite has been sent for Tuesday, June 18, from 11 a.m. to noon Eastern.
- [x]  It was sent from the CAIA A11y Capacity Calendar and appears on the calendar, to reflect our capacity to other teams. 
- [x] Updating the ticket now as well. Thanks!

**Comment by sara-amanda** [comment]
## Pilot Notes

- [x] Pilot completed 6/18/2024 by @SarahKay8 
- [x] Issues identified and documented in [DSVA Slack ](https://dsva.slack.com/archives/C044AGZFG2W/p1718903139209629?thread_ts=1718903082.604609&cid=C044AGZFG2W)^

> **During our CAIA Pilot we discovered a few issues:**
> 
> - The page <title> is incorrect and doesn’t match the H1 of the ‘form’. 
>     - It should say “Upload VA Form 21-0779” and right now it just says “Form upload”
> - AT users don’t know what’s expected from them when a page loads.  
>     - We’re not forcing focus to the right place. So AT-users won’t know where they are when continuing or re-loading a page

^ c/o @tbaker1026 in the [Form Upload Update Thread in the DSVA Slack channel: #veteran-facing-forms on 6/20/2024](https://dsva.slack.com/archives/C044AGZFG2W/p1718903082604609)

## Next Steps
- [x] Veteran Facing Forms Team is aware of the issues and is working on them. 
- [x] Closing out this ticket
- [x] [Refer to the main research ticket 85447](https://github.com/department-of-veterans-affairs/va.gov-team/issues/85447) and its task list for remaining research support.
